doozer/doozerlib/cli/config_plashet.py

```diff
@@ -535,7 +538,12 @@ def from_tags(config: SimpleNamespace, brew_tag: Tuple[Tuple[str, str], ...], em
             assembly = 'stream'
 
         # If assemblies are disabled, the true latest rpm builds from the tag will be collected; Otherwise we will only collect the rpm builds specific to that assembly.
-        tagged_builds = builder.from_tag(tag, inherit, assembly, event)
+        tagged_builds = builder.from_tag(tag, inherit, assembly, event=event, after_event=after_brew_event)
+        if not tagged_builds:
+            logger.info(f"Tag {tag} has no change for tagged rpms since --after-brew-event={after_brew_event}. "
+                        "Skipping plashet generation")
+            exit(0)
```

Comment:
I think we probably need a special exit code to signal the caller (pyartcd) not to update symlinks

Reply:
Good catch! I'll update it

Reply:
https://github.com/openshift-eng/art-tools/pull/476/commits/b3f312f2c09a4a90de567c6800e0684a3d10ec36